docs/feature_split_keyboard.md

```diff
@@ -130,24 +130,29 @@ To enable this method, add the following to your `config.h` file:
 #define EE_HANDS
 ```
 
-However, you'll have to flash the EEPROM files for the correct hand to each controller.  You can do this manually, or there are targets for avrdude and dfu to do this, while flashing the firmware: 
-
-* `:avrdude-split-left`
-* `:avrdude-split-right`
-* `:dfu-split-left`
-* `:dfu-split-right`
-* `:dfu-util-split-left`
-* `:dfu-util-split-right`
+Next, you will have to flash the EEPROM files once for the correct hand to the controller on each halve. You can do this manually with the following bootloader targets while flashing the firmware:
+
+* Pro micro stock controllers with Caterina bootloader:
+  * `:avrdude-split-left`
+  * `:avrdude-split-right`
+* Pro micro compatible controllers (like Elite-C) with DFU bootloader:
+  * `:dfu-split-left`
+  * `:dfu-split-right`
+* ARM controllers (like Proton-C) with DFU bootloader:
```

Comment:
Should probably note that you need to reset the eeprom after doing this, for dfu-util boards.

Reply:
I'm not familiar with split hardware using dfu-util board. Can you clarify the process for me to put that down in the description?

1. Flash with `:dfu-util-split-left`
2. Use QMK Toolbox to `Reset EEPROM` ?

Reply:
the dfu-util split command works by adding a define to specify left vs right, but this only gets set when you init the eeprom.  

Unfortunately, the toolbox can't be used for this.  You'd have to use the `EEP_RST` keycode, use bootmagic lite when flashing, or increment the eeconfig magic number. 

You can see this, here: 
https://github.com/qmk/qmk_firmware/blob/3dee6e28dd2fce27945bd464f5a80d3508e7c1d4/quantum/eeconfig.c#L60-L68

(eg, this only gets called on eeprom init/clear)

Reply:
Section updated with a note for `dfu-util`.